srcpkgs/system76-keyboard-configurator/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'system76-keyboard-configurator'
+pkgname=system76-keyboard-configurator
+version=1.3.10
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config glib-devel"
```

Comment:
this links against glib-devel, shouldn't it be in makedepends?

Reply:
Then it should be `libglib-devel` in makedepends I think. You would still need the `glib-devel` in hostmakedepends for cross-compiling as it tries to run `/usr/bin/glib-compile-resources`.

Reply:
Fixed.

Reply:
did you forget to push the fix?